inst/validation/specs.R

```diff
@@ -0,0 +1,28 @@
+# Use a list to declare the specs
+# nolint start
+specs_list <- list
+
+listing <- specs_list(
+  "display_listing" = "dv.listings displays a dataset in a tabular form",
+  "listing_selection" = "dv.listings includes a dropdown menu to select which listing to be shown.",
+  "listings_label" = "dv.listings displays the label of a listing if available. The label is concatenated to the listing’s dataset name and the resulting strings are provided as choices in the listings dropdown menu.",
+  "column_selection" = "dv.listings includes a dropdown menu to select the columns from the selected listing to be shown and arrange their order.",
+  "column_label" = "dv.listings displays the column labels of a listing if available. Column names are pasted together with their label. These extended column titles replace the original column names, so that they are visible in the listings display and column dropdown menu.",
+  "sorting_columns" = "dv.listings includes sorting functionality for each of the different variables included in the dataset",
+  "restore_row_order" = "dv.listings includes a button to restore the row order of a listing to the state as it is in the original data.",
+  "default_vars" = "If pre-specifications for default columns are available, dv.listings will display them at app launch for the respective listing. If not, dv.listings will show the first six columns of the listing - or all columns, in case the number of columns is less than six.",
+  "retain_last_selection" = "dv.listings can remember and retain the last column selections after switching listings during the current session. It also restores the ",
```

Comment:
it also restores the...?

Reply:
Oh must have missed it. Will complete the sentence. :)